.github/actions/upstream-test/action.yml

```diff
@@ -44,13 +44,7 @@ runs:
         uses: firehed/multistage-docker-build-action@v1
         with:
           repository: >
-            ghcr.io/${{ github.repository_owner }}/
-            dagmc-ci-ubuntu-${{ inputs.ubuntu_version }}
-            -${{ inputs.compiler}}
-            -hdf5_${{ inputs.hdf5_version}}
-            -moab_${{ inputs.moab_version }}
-            -geant4_${{ inputs.geant4_version }}
-            -double_down_${{ inputs.double_down_version }}
+            ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ inputs.ubuntu_version }}-${{ inputs.compiler}}-hdf5_${{ inputs.hdf5_version}}-moab_${{ inputs.moab_version }}-geant4_${{ inputs.geant4_version }}-double_down_${{ inputs.double_down_version }}
```

Comment:
```suggestion
            ghcr.io/${{ github.repository_owner 
            }}/dagmc-ci-ubuntu-${{ inputs.ubuntu_version 
            }}-${{ inputs.compiler
            }}-hdf5_${{ inputs.hdf5_version
            }}-moab_${{ inputs.moab_version 
            }}-geant4_${{ inputs.geant4_version 
            }}-double_down_${{ inputs.double_down_version 
            }}
```

Reply:
However, we need to give this a test run, don't know if the action actually works in this way.

Reply:
Or, I think we may proceed with the following code that we already used in our `docker_publish.yml`:

```yml
      - name: Configure docker image tag
        run: |
          image_base_tag=ghcr.io/${{ github.repository_owner }}/
          image_base_tag+=dagmc-ci-ubuntu-${{ matrix.ubuntu_version }}
          image_base_tag+=-${{ matrix.compiler}}
          image_base_tag+=-hdf5_${{ matrix.hdf5_version}}
          image_base_tag+=-moab_${{ matrix.moab_version }}
          image_base_tag+=-geant4_${{ matrix.geant4_version }}
          image_base_tag+=-double_down_${{ matrix.double_down_version }}
          echo "image_base_tag=${image_base_tag}" >> "$GITHUB_ENV"
```

Reply:
If you make a PR with this update, I'll withdraw mine and review yours.

Reply:
PR #949 fixed the incorrect and introduced illegal spaces in the `action.yml` file.